packages/connect-iframe/src/index.ts

```diff
@@ -55,6 +60,15 @@ const handleMessage = (event: PostMessageEvent) => {
         postMessage(createPopupMessage(POPUP.CANCEL_POPUP_REQUEST));
     };
 
+    if (data.type === 'ADD_LOG' && data.payload.prefix === '@trezor/connect-web') {
+        const { level, prefix, message } = data.payload;
+        if (Array.isArray(message)) {
+            connectWebLog.addMessage(level, prefix, ...message);
+        } else {
+            connectWebLog.addMessage(level, message);
+        }
```

Comment:
shoudln't it have `return` statement here?
otherwise this messages bubbles down to core
<img width="561" alt="image" src="https://github.com/trezor/trezor-suite/assets/30367552/1c463767-90e7-4cef-8350-10d4c34a6c49">

Reply:
Sure, yes, I fixed it. Thanks.